cloud/amazon/rds.py

```diff
@@ -42,7 +42,7 @@
       - The type of database.  Used only when command=create.
```

Comment:
No problem, sorry for not made it clear:

The documentation should look like:
~~~
  db_engine:
    description:
      - The type of database. Used only when command=create.
      - mariadb was added in version 2.2
    required: false
    default: null
    choices: [ 'mariadb', 'MySQL', 'oracle-se1', 'oracle-se', 'oracle-ee', 'sqlserver-ee', 'sqlserver-se', 'sqlserver-ex', 
~~~